releases/net472/dotnet472-changes.md

```diff
@@ -219,3 +221,6 @@
 * Fixed a problem where a WPF application that changes TreeView.IsEnabled and (at the same time) changes the underlying collection(s) could experience a spurious ElementNotAvailableException. [581149, PresentationCore.dll, PresentationFramework.dll, Bug]
 * Fixed a problem where a WPF application can crash with NullReferenceException when changing a property used by a DataTrigger whose host element has been removed from the visual tree.  This crash only occurs if the host element is garbage-collected during a small window of time during the property-change notification. [581151, PresentationCore.dll, Bug]
 * Fixed a problem that causes XAML Browser Applications (XBAP’s) targeting .NET 3.5 to sometimes be loaded using .NET 4.x runtime incorrectly. This fix requires the setting of an opt-in registry key described at https://github.com/Microsoft/dotnet/blob/net472/releases/net472/KnownIssues/560016-XBAP%20Applications%20targeting%20.NET%202.0%2C3.5%20select%20wrong%20runtime%20version.md [581487, PresentationHost_v0400.dll, Bug]
+* Adds an AppContext switch that opts out of some work done during AppDomain or process shutdown. This can reduce (but not eliminate) the chance of a crash in apps that make unwarranted assumptions about the timing of the finalizer thread. [593963, WindowsBase.dll, Bug]
+* Fixes a crash in WPF when replacing multiple characters with a single character (in a different language than replaced text) using IMEPad. [605996, PresentationFramework.dll, Bug]
```

Comment:
than replaced text ==> than the replaced text

Reply:
Will fix